datafn/python/datafn/handlers/mutation.py

```
@@ -0,0 +1,854 @@
+            await run_after_hook(plugins, "server", "after_mutation", ctx, payload, response, logger=logger)
+
+        if logger:
+            logger.debug("Mutation completed", {"resource": resource})
+
+        return response
+
+    except DatafnError as e:
+        if logger:
+            logger.error("Mutation failed", {"code": e.code, "message": e.message})
+        return e.to_envelope()
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)